cmd/minishift/cmd/root.go

```diff
@@ -103,6 +103,10 @@ var RootCmd = &cobra.Command{
 			atexit.ExitWithMessage(1, fmt.Sprintf("Error creating config for VM: %s", err.Error()))
 		}
 
+		// Persist instance kube config path to be used by other commands
+		minishiftConfig.InstanceConfig.KubeConfigPath = constants.KubeConfigPath
```

Comment:
After taking a better look, we should us `constants.KubeConfigPath` where ever `minishiftConfig.InstanceConfig.KubeConfigPath` is used currently in the PR. Both of their purpose is the same thing.  IMO `constants.KubeConfigPath` is better solution than InstanceConfig.KubeConfigPath. 
@hferentschik WDYT?

Reply:
Agree @LalatenduMohanty , same can be achieved with `constants.KubeConfigPath`. Changes done.